### DISCUSSION TITLE
How do i change the colour of the bounding boxes in yolov8?

### DISCUSSION THREAD

**Discussion by Sankie005** [ultralytics/ultralytics#2866] [Q&A]
Hey there! I've been using yolov8 for a while now and i'd really like to be able to customize the look of the bounding boxes with things like line thickness, colour and if possible rounded edges. It's a minor thing but i think it would help my projects be a lot more presentable, i've tried searching for this online but i could only find answers pertaining to an older build, I'd be really grateful if you were able to help me out.

**Comment by glenn-jocher** [comment]
@Sankie005 hello! I'm glad to hear you're using YOLOv8 for your projects. 😊 Customizing the appearance of bounding boxes, such as color and line thickness, can be done by modifying the visualization code within the repository.

For YOLOv8, you'll want to look into the code responsible for drawing the bounding boxes, typically found in the files that handle the prediction outputs. You can adjust the parameters for color and line thickness there. Rounded edges are a bit more complex and would require additional custom drawing functions.

Remember, any modifications will need to adhere to the visualization functions' structure. For more detailed guidance on the different modes and tasks that YOLOv8 supports, you can refer to our documentation at https://docs.ultralytics.com.

Happy customizing, and I hope your projects turn out as vibrant and presentable as you envision! 🎨🖼️

**Reply by Sankie005** [reply]
Thank-you for your response! I was able to get it working! 🎊

**Reply by pderrenger** [reply]
@Sankie005 that's fantastic to hear! 🥳 I'm thrilled that you were able to customize the bounding boxes to your liking. If you have any more questions or need further assistance as you continue working on your projects, don't hesitate to reach out. Keep up the great work with YOLOv8! 🚀
